src/application/services/useNote.ts

```diff
@@ -147,7 +147,7 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     if (!firstNoteBlock || !Boolean(firstNoteBlock.data.text)) {
```

Comment:
it would be better if we will reuse the `getTitle()` util here

Reply:
I think there will be an issue of using I18n at the top of a `setup` function.

Reply:
you can pass a `defaultTitle` as an optional argument, for example